Ollivanders/src/net/pottercraft/ollivanders2/stationaryspell/HARMONIA_NECTERE_PASSUS.java

```diff
@@ -102,7 +102,7 @@ public void checkEffect ()
     */
    private boolean cabinetCheck (Block feet)
    {
-      if (feet.getType() != Material.AIR && feet.getType() != Material.WALL_SIGN)
+      if (feet.getType() != Material.AIR && common.materialIsASign(feet.getType()))
```

Comment:
Should be not materialIsASign.  Also is it important that we're checking for any sign not a wall sign?

Reply:
made the check for wall signs specifically

Reply:
fixed